demos/bartlett1932/__init__.py

```diff
@@ -0,0 +1,7 @@
+from dallinger import db
+from experiment import Bartlett1932
+
+
+def build():
+    """Potentially move PsiturkConfig stuff in here"""
+    return Bartlett1932(db.init_db())
```

Comment:
@jessesnyder can comment on the intent of this, I'm not sure what it's for either.